clients/src/test/java/org/apache/kafka/common/message/MessageTest.java

```diff
@@ -465,13 +465,25 @@ public void testOffsetFetchVersions() throws Exception {
         OffsetFetchRequestData allPartitionData = new OffsetFetchRequestData()
                                                       .setGroupId(groupId)
                                                       .setTopics(null);
+
+        OffsetFetchRequestData requireStableData = new OffsetFetchRequestData()
+                                                       .setGroupId(groupId)
+                                                       .setTopics(null)
+                                                       .setRequireStable(true);
+
         for (short version = 0; version <= ApiKeys.OFFSET_FETCH.latestVersion(); version++) {
+            final short finalVersion = version;
             if (version < 2) {
-                final short finalVersion = version;
                 assertThrows(SchemaException.class, () -> testAllMessageRoundTripsFromVersion(finalVersion, allPartitionData));
             } else {
                 testAllMessageRoundTripsFromVersion(version, allPartitionData);
             }
+
+            if (version < 7) {
+                assertThrows(UnsupportedVersionException.class, () -> testAllMessageRoundTripsFromVersion(finalVersion, requireStableData));
```

Comment:
I thought we were not throwing anymore?

Reply:
It's thrown from the OffsetFetchRequestData:
`Attempted to write a non-default requireStable at version ...`

Reply:
In the Builder we revert to `false` if the version is not high enough. I think we should make the schema definition consistent with this behavior. We can set `requireStable` as ignorable.